SPECS-EXTENDED/rlwrap/rlwrap.spec

```diff
@@ -0,0 +1,73 @@
+Name:           rlwrap
+Version:        0.46.1
+Release:        1%{?dist}
+Summary:        Wrapper for GNU readline
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/hanslub42/rlwrap
+Source:         https://github.com/hanslub42/rlwrap/releases/download/%{version}/%{name}-%{version}.tar.gz
+
```

Comment:
Would you please mind addressing the `linter` suggestions as well.

Reply:
done